File: app/src/main/java/eu/toma/dev/playground/warcraft/model/item/ItemSource.java
package eu.toma.dev.playground.warcraft.model.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class ItemSource {

    @SerializedName("sourceId")
    @Expose
    public Integer sourceId;
    @SerializedName("sourceType")
    @Expose
    public String sourceType;

}

File: app/src/main/kotlin/warcraft/model/item/ItemSource.kt
package warcraft.model.item

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class ItemSource {

    @SerializedName("sourceId")
    @Expose
    var sourceId: Int? = null
    @SerializedName("sourceType")
    @Expose
    var sourceType: String

}
